stories/TextSquare.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react'
+
+import { storiesOf } from '@storybook/react'
+import { action } from '@storybook/addon-actions'
+import { boolean } from '@storybook/addon-knobs'
+
+import TextSquare from 'components/TextSquare'
+import NoWallet from 'components/NoWallet'
+
+const CenterDecor = (story: Function) => (
+  <div
+    style={{
+      display: 'flex',
+      height: '100vh',
+      justifyContent: 'center',
+      alignItems: 'center',
+    }}
+  >
+    {story()}
+  </div>
+)
+
+storiesOf('TextSquare', module)
+  .addDecorator(CenterDecor)
+  .addWithJSX('TextSquareLeft[Intro]', () => 
+    <section className="home">
```

Comment:
`<section> can be added once as a decorator, before `CenterDecor``

Reply:
But then you'll have to do
```
.addWithJSX('TextSquareBoth', (): any =>
    [<TextSquare />, <NoWallet handleClick={action('ButtonCTA clicked')} hide={boolean('Hide/Show', false)} />],
)
```
So it's your call

Reply:
Thanks - had it as a decorator before but changed it back instead of going the [] approach. Will most likely change to keep DRY - cheers